docs-v2/docs.json

```diff
@@ -157,6 +104,84 @@
           }
         ]
       },
+      {
+        "anchor": "Implementation Guides",
+        "icon": "pen",
+        "groups": [
+          { 
+            "group": "API Auth",
+            "pages": [
+              "implementation-guides/api-auth/configure-integration",
+              "implementation-guides/api-auth/implement-api-auth",
+              "implementation-guides/api-auth/customize-connect-ui"
+            ]
+          },
+          { 
+            "group": "Syncs",
+            "pages": [
+              "implementation-guides/syncs/implement-a-sync",
```

Comment:
<img width="398" height="478" alt="CleanShot 2025-09-10 at 16 07 10@2x" src="https://github.com/user-attachments/assets/0ccc3298-69b2-4905-9fc0-97d294f61d52" />

I would try to be consistent with: 
1. Casing
2. Verb vs. name titles

Here: 
- Implement a sync
- Detect deletions
- Sync in real-time
- Handle large datasets
- Partition syncs

Other sections sections also have casing inconsistencies.